internal/app/unifi/cmd/auth_command.go

```diff
@@ -2,9 +2,9 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/gsdevme/unifi/pkg/unifi/apiclient"
```

Comment:
If this was me, I'd drop the `unifi` from`pkg/unifi/apiclient` you already have `unifi` in the name, so it makes sense that `apiclient` would be for `unifi`?